src/components/content/content-blocks/block-quote/block-quote.st.css

```diff
@@ -1,3 +1,7 @@
 @namespace "BlockQuote";
 
 .root {}
+
+.code {
+	white-space: pre;
```

Comment:
Why do we need this style both in `BlockQuote` and in `CodeBlock`?

Reply:
Remains from a refactoring attempt. Reverted